### PR TITLE
Removes Unused Xenoarch Code

### PR DESCRIPTION
**Pull request by Fox-McCloud**
This was going to be done a long time ago, but there was so many map changes associated with it, that I held off on it.

Either case, Xenoarch has been dead for a long time now, and now, with the removal of the science outpost, there's nothing that is utilizing these files anymore (thank goodness).

Yay, Cael code bye bye.

Only thing worth documenting is the removal of a single stasis book from the medical break room.

:cl: Fox McCloud
tweak: removed a stasis book from the medical break room
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/shards.dm [hunk 1]
```diff
@@ -87,4 +87,42 @@
...
 					H.UpdateDamageIcon()
 				H.updatehealth()
-	..()
\ No newline at end of file
+	..()
+
+/obj/item/weapon/shard/plasma
+	name = "plasma shard"
+	desc = "A shard of plasma glass. Considerably tougher then normal glass shards. Apparently not tough enough to be a window."
+	force = 8
```
Comment (SamHPurp) [inline code/game/objects/items/weapons/shards.dm]: Kinda wish these did some toxin dmg too

File: code/game/objects/structures/misc.dm [hunk 2]
```diff
@@ -121,3 +121,12 @@
 	if(last_ghost_alert + ghost_alert_delay < world.time)
 		notify_ghosts("[src] active in [get_area(src)].", 'sound/effects/ghost2.ogg', title = alert_title, source = attack_atom, action = (attack_atom == src ? NOTIFY_JUMP : NOTIFY_ATTACK))
 		last_ghost_alert = world.time
+
+/obj/structure/boulder
+	name = "boulder"
+	desc = "A large rock."
```
Comment (SamHPurp) [inline code/game/objects/structures/misc.dm]: thats a nice boulder.

Comment (Citinited) [inline code/game/objects/structures/misc.dm]: "This is a gneiss boulder." 🤔

Comment (owenowen212) [inline code/game/objects/structures/misc.dm]: Don't take this for granite.

Comment (Birdtalon) [inline code/game/objects/structures/misc.dm]: I think this discussion has got off to a rocky start.

Comment (Fox-McCloud) [inline code/game/objects/structures/misc.dm]: *scream

**Comment by Crazylemon64** [comment]
🔥 🔥 🔥

**Pull request merged**